docs/docs/user/core-concepts/operation-mode.md

```diff
@@ -0,0 +1,136 @@
+# Operation Modes in KPOps
+
+KPOps supports three operation modes—`managed`, `manifest`, and `argo`. These modes determine how resources are managed and allow users to tailor their deployment strategy.
+
+- **Managed Mode** (default): KPOps uses helm under the hood to manage the installation of applications in the defined pipeline.yaml.
```

Comment:
```suggestion
- **Managed Mode** (default): KPOps uses Helm under the hood to manage the installation of applications in the defined pipeline.yaml.
```

Reply:
not just Helm, we talk to all the necessary services and deploy directly to the cluster. That's the main difference to Manifest mode, which also uses Helm under the hood, but only for `helm template`.